src/english_confusables.rs

```diff
@@ -46,7 +46,7 @@ pub fn map(input: String) -> String {
 
     confusables.insert('R', vec!['\u{211B}','\u{211C}','\u{211D}','\u{1D411}','\u{1D445}','\u{1D479}','\u{1D4E1}','\u{1D57D}','\u{1D5B1}','\u{1D5E5}','\u{1D619}','\u{1D64D}','\u{1D681}','\u{01A6}','\u{13A1}','\u{13D2}','\u{1587}','\u{A4E3}']);
 
-    confusables.insert('S', vec!['\u{FF33}','\u{1D412}','\u{1D446}','\u{1D47A}','\u{1D4AE}','\u{1D4E2}','\u{1D516}','\u{1D54A}','\u{1D57E}','\u{1D5B2}','\u{1D5E6}','\u{1D61A}','\u{1D64E}','\u{1D682}','\u{0405}','\u{054F}','\u{13D5}','\u{13DA}','\u{A4E2}','\u{10296}','\u{10420}']);
+    confusables.insert('S', vec!['\u{FF33}','\u{1D412}','\u{1D446}','\u{1D47A}','\u{1D4AE}','\u{1D4E2}','\u{1D516}','\u{1D54A}','\u{1D57E}','\u{1D5B2}','\u{1D5E6}','\u{1D61A}','\u{1D64E}','\u{1D682}','\u{0405}','\u{054F}','\u{13D5}','\u{13DA}','\u{A4E2}','\u{10296}','\u{10420}', '$']);
```

Comment:
Please use unicode constant `\u{0024}`. This is so that, ironically, maintainers don't get the character in the source code confused with other similar-looking Unicode characters.

Speaking of which, we may want to also consider including 
U+FE69  ﹩ 
U+FF04  ＄ 
U+1F4B2  💲